charts/rbs/templates/proxy-apache-httpd-configmap.yaml

```diff
@@ -0,0 +1,83 @@
+{{- if .Values.proxy.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: terra-rbs-proxy-configmap
+  labels:
+    {{ template "rbs.labels" . }}
+data:
+  apache-httpd-proxy-config: |-
+    ServerAdmin ${SERVER_ADMIN}
+    ServerName {{ template "rbs.fqdn" . }}
+    ServerTokens ProductOnly
+    TraceEnable off
+
+    LogFormat "%h %l %u \"%{OIDC_CLAIM_email}i\" \"%{X-App-ID}i\" [%{%FT%T}t.%{msec_frac}t%{%z}t] %D %X \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\"" combined
+    LogFormat "%{X-Forwarded-For}i %l %u \"%{OIDC_CLAIM_email}i\" \"%{X-App-ID}i\" [%{%FT%T}t.%{msec_frac}t%{%z}t] %D %X \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\"" proxy
+    SetEnvIf X-Forwarded-For "^.*\..*\..*\..*" forwarded
+    CustomLog "/dev/stdout" combined env=!forwarded
+    CustomLog "/dev/stdout" proxy env=forwarded
+
+    LogLevel {{ .Values.proxy.logLevel }}
+
+    Header unset X-Frame-Options
+    Header always set X-Frame-Options SAMEORIGIN
+
+    ProxyTimeout ${PROXY_TIMEOUT}
+    OIDCOAuthTokenIntrospectionInterval 60
+
+    <VirtualHost _default_:${HTTPD_PORT}>
+      ErrorLog /dev/stdout
+      CustomLog "/dev/stdout" combined
+      Redirect / https://{{ template "rbs.fqdn" . }}/
+    </VirtualHost>
+
+    <VirtualHost _default_:${SSL_HTTPD_PORT}>
+        ServerAdmin ${SERVER_ADMIN}
+        ServerName {{ template "rbs.fqdn" . }}
+
+        SSLEngine On
+        SSLProxyEngine On
+        SSLCertificateFile /etc/ssl/certs/server.crt
+        SSLCertificateKeyFile /etc/ssl/private/server.key
+        {{ if .Values.vaultCert.enabled -}}
+          SSLCertificateChainFile /etc/ssl/certs/ca-bundle.crt
+        {{- end }}
+
+        ErrorLog /dev/stdout
+        CustomLog "/dev/stdout" combined
+
+        <Location ${PROXY_PATH}>
+          RewriteCond %{REQUEST_METHOD} OPTIONS
+          RewriteRule ^(.*)$ $1 [R=204,L]
+
+          <Limit OPTIONS>
+              Require all granted
+          </Limit>
+
+          ${AUTH_TYPE}
+          ${AUTH_REQUIRE}
+
+          ProxyPass http://localhost:8080/
+          ProxyPassReverse http://localhost:8080/
+        </Location>
+
+        <Location ${PROXY_PATH2}>
+          RewriteEngine On
+          RewriteCond %{REQUEST_METHOD} OPTIONS
+          RewriteRule ^(.*)$ $1 [R=204,L]
+
+          <Limit OPTIONS>
+              Require all granted
+          </Limit>
+
+          ${AUTH_TYPE2}
+          ${AUTH_REQUIRE2}
+
+          ProxyPass http://localhost:8080/api
+          ProxyPassReverse http://localhost:8080/api
+
+          AddOutputFilterByType DEFLATE application/json text/plain text/html application/javascript application/x-javascript
+        </Location>
+    </VirtualHost>
+{{- end }}
```

Comment:
We need to use the updated [apache config](https://github.com/broadinstitute/terra-helm/blob/master/charts/workspacemanager/templates/proxy-apache-httpd-configmap.yaml) that was added for workspace manager recently in this case too. There were some security issues that came up with the infosec team when reviewing this config. @gmalkov might have a little more context on this.

Reply:
Thanks Mike! I will check with Greg. But for RBS, there is no user facing APIs, so I don't think we will need Ldap stuff, AoU precondition. Not sure the Tcell thing though

Reply:
Hi Mike, checked with Greg offline. RBS probably need the same stuff as WM. We will live the RBS as it, and switch later if we need.